pkg/plugins/manager/signature/authorizer.go

```diff
@@ -34,5 +34,13 @@ func (u *UnsignedPluginAuthorizer) CanLoadPlugin(p *plugins.Plugin) bool {
 		}
 	}
 
+	if p.Parent != nil {
```

Comment:
Would it be useful to include a comment saying why we do this? Maybe something like "This treats the children of a whitelisted plugin as also being whitelisted."